recipes/trackhub/0.1.2019.12.24/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "trackhub" %}
+{% set version = "0.1.2019.12.24" %}
+{% set commit = "157ac4d3c392bd8c47b675a68f3c1a25b08a4afa" %}
+{% set sha256 = "a13b59e942d94bd6bdcb95c6cd480717224641f8e66400581d4ffa3bc7732874" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # "stale" PR here: https://github.com/daler/trackhub/pull/33
+  url: https://github.com/daler/trackhub/archive/{{ commit }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: $PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
This can be `script: "{{ PYTHON }} -m pip install . --no-deps -vv"`, https://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=python#use-pip.